.gitattributes

```diff
@@ -8,3 +8,4 @@
 *.css  diff=css
 *.html diff=html
 *.erb  diff=html
+/CHANGELOG.md merge=union
```

Comment:
This allows changelog entries not to cause merge conflicts. The idea is that if a merge conflict would look like this:

```diff
<<<<<<<<
+- Add something
========
+- Do something else
>>>>>>>>
```

The merge conflict would then take the _union_ of the changes, which is basically just removing the merge-conflict markers:

```diff
+- Add something
+- Do something else
```

Notice these two commits from other branches merged into this branch one after the other — neither one acknowledges the other, but notice the merge commit from the second one:
- 4e459997bc322db548b921d0fcfd2f736b550232
- fe59da0860cad92156721615e1dab077796c6396
  - The [merge commit from this one](https://github.com/forem/forem/pull/13750/commits/066e7c317601d774296558674bf5740f23e750d0)

Reply:
Nice one @jgaskins :-)